cross/he853/Makefile

```diff
@@ -0,0 +1,22 @@
+PKG_NAME = he853-cli
+PKG_VERS = 1.0.1
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/hphde/$(PKG_NAME)/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+BUILD_DEPENDS = cross/libusb
+
+HOMEPAGE = https://github.com/hphde/$(PKG_NAME)
+COMMENT  = HomeEasy HE853 USB device executable 
+LICENSE  = GPLv2
+
+CONFIGURE_TARGET = none
+
+INSTALL_TARGET = myInstall
+
+include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: myInstall
+myInstall:
+	$(RUN) $(MAKE) install PREFIX=$(STAGING_INSTALL_PREFIX)
```

Comment:
I can't compile due to the Makefile of HE853 file which try to install into `/lib`.
Replace by this line will install the locally

    mkdir -p $(STAGING_INSTALL_PREFIX)/lib/udev/rules.d
    $(RUN) $(MAKE) install PREFIX=$(STAGING_INSTALL_PREFIX) MY_UDEV_RULES=$(STAGING_INSTALL_PREFIX)/lib/udev/rules.d